src/Features/LanguageServer/Protocol/Handler/Symbols/WorkspaceSymbolsHandler.cs

```diff
@@ -8,65 +8,104 @@
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Editor.Implementation.NavigateTo;
+using Microsoft.CodeAnalysis.Editor.Shared.Extensions;
 using Microsoft.CodeAnalysis.Host.Mef;
 using Microsoft.CodeAnalysis.NavigateTo;
+using Microsoft.CodeAnalysis.Shared.TestHooks;
 using Microsoft.VisualStudio.LanguageServer.Protocol;
+using Microsoft.VisualStudio.Text.Adornments;
 using Roslyn.Utilities;
 
 namespace Microsoft.CodeAnalysis.LanguageServer.Handler
 {
     [Shared]
     [ExportLspMethod(Methods.WorkspaceSymbolName, mutatesSolutionState: false)]
-    internal class WorkspaceSymbolsHandler : IRequestHandler<WorkspaceSymbolParams, SymbolInformation[]>
+    internal class WorkspaceSymbolsHandler : IRequestHandler<WorkspaceSymbolParams, SymbolInformation[]?>
```

Comment:
unified the logic here between the liveshare handler (this) and the previous logic in the OOP language server
http://sourceroslyn.io/#Microsoft.CodeAnalysis.Remote.ServiceHub/Services/LanguageServer/LanguageServer.cs,87